charts/tfy-notebook-controller/values.yaml

```
@@ -76,7 +76,7 @@ notebookBaseDomainUrl: <to_be_provided>
 ## @param oauth.hmac HMAC key used for encoding/decoding tokens in OAuth2. This key is crucial for maintaining token integrity and security.
 ##
 oauth:
-  enabled: false
+  enabled: true
```

```suggestion
  enabled: false
```

Let us not enable this by default as of now. @srihari-tf